.github/workflows/test-past-releases.yml

```diff
@@ -118,15 +76,13 @@ jobs:
       - name: Install SCT (Windows)
         if: runner.os == 'Windows'
         shell: cmd
-        run: | 
-          # Add fix for clash between pip 24.1 and dipy 1.5.0
-          install_sct.bat
+        run: install_sct.bat
 
       - name: Update environment variables
         run: |
           if [ "$RUNNER_OS" == "Windows" ]; then
             # SCT_DIR was different in the first iteration of Windows support
-            if [ "${{ matrix.sct_version }}" == "5.7" ] || [ "${{ matrix.sct_version }}" == "5.8" ]; then
+            if [ "${{ matrix.sct_version }}" == "5.7-bp" ] || [ "${{ matrix.sct_version }}" == "5.8-bp" ]; then
               export SCT_DIR="D:\a\spinalcordtoolbox\spinalcordtoolbox"
             else
               export SCT_DIR="$USERPROFILE\sct_${{ matrix.sct_version }}"
```

Comment:
Silly me, even though the branch names include this `-bp` slug, the actual `SCT_DIR` will **not** include the `-bp` slug, since it's based off of the version in `version.txt`.

Reply:
Resolved by https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4780/commits/790f47298e0a756a9eeb881b88af5ac5dfb69236.